tools/lexLog_tools.mll

```diff
@@ -234,18 +235,19 @@ and pline bds fs abs = parse
 | blank* fault blank*
     '(' blank* ('P'? (num as proc)) (':' (name as lbl))? blank* ','
      ((loc|new_loc) as loc) (':' alpha+)? blank* (* skip optional tag *)
+     (',' blank* (("kvm")|(fault_type as ftype)))? blank*
```

Comment:
Nice trick. Unfortunately it does not function because the "|" is non-determimistic and that teh string "kvm" matches both alternatives. The same idea can be implemented differently by inserting this code in action:
```OCaml
let ftype = match ftype with Some "kvm" -> None | _ -> ftype in
```